storage/jellyfish_merkle/src/mock_tree_store.rs

```diff
@@ -46,12 +46,13 @@ impl MockTreeStore {
     }
 
     pub fn write_tree_update_batch(&self, batch: TreeUpdateBatch) -> Result<()> {
-        let (node_batch, retired_record_batch) = batch.into();
```

Comment:
Do you need an .into() implementation anymore or was this the only use?

Reply:
now the batch has 4 fields so I think it is better to avoid `.into()`